src/bootstrap.tsx

```diff
@@ -43,14 +50,62 @@ loadingMessage.textContent = 'Initializing Cytoscape. Please wait...'
 document.body.appendChild(loadingMessage)
 
 const keycloak = new Keycloak(keycloakConfig)
+
+const handleVerify = async () => {
+  await keycloak.loadUserProfile()
+  window.location.reload()
+}
+
+const handleCancel = () => {
+  keycloak.logout()
+}
+
+// Function to check if the user's email is verified
+const checkUserVerification = async () => {
+  try {
+    const ndexClient = new NDEx(appConfig.ndexBaseUrl)
+    ndexClient.setAuthToken(keycloak.token)
+    await ndexClient.getSignedInUser()
+    return {
+      isVerified: true,
```

Comment:
In my opinion, this function should always return a consistent object.  i.e. always return a object with `isVerified`, `userName` and `userEmail`, even if the the fields will not always be used by the calling function.